lib/capybara/mechanize/version.rb

```diff
@@ -2,6 +2,6 @@
 
 module Capybara
   module Mechanize
-    VERSION = '1.11.0'
+    VERSION = '1.12.0'
```

Comment:
This actually drops support for capaybara 2.x, we probably should have done a major version bump here.